integration-tests/models/test_flash_llama_marlin.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+
+@pytest.fixture(scope="module")
+def flash_llama_marlin_handle(launcher):
+    with launcher(
+        "neuralmagic/llama-2-7b-chat-marlin", num_shard=2, quantize="marlin"
```

Comment:
Why using llama2 instead of llama3 (or other new models?)

Reply:
Mostly because it was the only neuralmagic llama model, I'll find something else for the next PR.